catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestBaseConvert.java

```diff
@@ -120,6 +133,59 @@ protected static Types.NestedField[] createNestedField(String... colNames) {
     return results.toArray(new Types.NestedField[0]);
   }
 
+  protected static String convertIcebergTransform(SortField sortField) {
+    String transform = sortField.transform().toString();
+    if (transform.startsWith("year")
+        || transform.startsWith("month")
+        || transform.startsWith("day")
+        || transform.startsWith("hour")
+        || transform.startsWith("identity")) {
+      return transform + "(";
+    } else if (transform.startsWith("truncate") || transform.startsWith("bucket")) {
+      return transform.replace("[", "(").replace("]", ", ");
+    } else {
+      throw new RuntimeException("Unsupported Iceberg transform type");
+    }
+  }
+
+  protected static String convertGravitinoSortOrderExpression(Expression sortOrderExpression) {
```

Comment:
rename to `getGravitinoSortOrderExpressionString`?

Reply:
updated.